examples/data-sources/pingfederate_virtual_host_names/resource.tf

```diff
@@ -0,0 +1,7 @@
+resource "pingfederate_virtual_host_names" "myVirtualHostNamesExample" {
```

Comment:
The example should only include the data source block

Reply:
done